### PR TITLE
wrj-04-29

### PR DESCRIPTION
**Pull request by ailntps**

### CODE REVIEW

**Comment by ehds** [comment]
Please add this item into  the [directory listing](https://github.com/CDDSCLab/Weekly-Group-Meeting-Paper-List/blob/main/README.md).

**Review by ehds — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -19,3 +19,5 @@ DSCL weekly group meeting paper list.
 | 2022-03-04 | NVM for Hashtables | NVM, Hashtables | [doc](./meeting-summary/2022-03-04-面向NVM的数据库哈希索引优化研究.md) |
 | 2022-03-25 | Memory Database | Trie, ART, HOT, Hyperion | [doc](./meeting-summary/2022-03-25-内存数据库简述及索引优化.md) |
 | 2022-04-01 | HTAP, Data formate | HTAP, PAX, Hyper, FSM, RealTime LSM | [doc](./meeting_summary/2022-04-01-HTAP概述及存储格式.md) |
+| 2022-04-29 | stream system rescale | Flink,Chi,TImely dataflow,megaPhone | [doc](./meeting-summary/2022-04-29-流计算引擎弹性扩展综述.md) |
```
Comment (IcePigZDB) [inline README.md:22]:
```suggestion
| 2022-04-29 | Stream System Rescale | Flink, Chi, TImely Dataflow, MegaPhone | [doc](./meeting-summary/2022-04-29-流计算引擎弹性扩展综述.md) |
```

File: meeting-summary/2022-04-29-流计算引擎弹性扩展综述.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
+# 2022.04.29 分享纪要
+
+- 分享人：王仁杰
+- 记录人：王仁杰
+- 关键词：**Flink，rescale，chi**
```
Comment (IcePigZDB) [inline meeting-summary/2022-04-29-流计算引擎弹性扩展综述.md:5]:
```suggestion
- 关键词：**Flink，Rescale，Chi**
```

**Review by IcePigZDB — APPROVED** [review]
LGTM

**Pull request merged**